.github/workflows/pkgcheck.yml

```diff
@@ -8,14 +8,37 @@ jobs:
       fail-fast: false
       matrix:
         name: [
-          Ubuntu GCC
+          Ubuntu GCC,
+          Ubuntu GCC AARCH64,
+          Ubuntu GCC PPC
```

Comment:
@dankegel Shouldn't you add `Ubuntu GCC PPC64LE` to this list too?

Reply:
yep.